src/hotspot/share/gc/g1/g1ConcurrentMarkThread.cpp

```diff
@@ -139,8 +139,8 @@ void G1ConcurrentMarkThread::run_service() {
     assert(in_progress(), "must be");
 
     GCIdMark gc_id_mark;
-    GCTraceConcTime(Info, gc) tt(FormatBuffer<128>("Concurrent %s Cycle",
-                                                   _state == FullMark ? "Mark" : "Undo"));
+    FormatBuffer<128> _format("Concurrent %s Cycle", _state == FullMark ? "Mark" : "Undo");
+    GCTraceConcTime(Info, gc) tt(_format);
```

Comment:
```suggestion
    FormatBuffer<128> title("Concurrent %s Cycle", _state == FullMark ? "Mark" : "Undo");
    GCTraceConcTime(Info, gc) tt(title);
```
Leading underscores are reserved for members of structs and classes in Hotspot. I suggested `title` here, but `format` would be as good.

Apart from that lgtm. Thanks for investigating this.

Reply:
Was just about to make the same comment :)

Reply:
Thanks Tomas for the review, will update with title